site/app/controllers/MiscController.php

```diff
@@ -179,19 +179,20 @@ private function displayFile() {
             return false;
         }
 
-        $mime_type = FileUtils::getMimeType($_REQUEST['path']);
+		$corrected_name = pathinfo($_REQUEST['path'], 1) . "/" . rawurlencode( basename($_REQUEST['path']));
```

Comment:
Fix spacing to be consistent with rest of file. Use a PHP constant for the option flag instead of a magic 1.